mlem/contrib/docker/dockerfile.j2

```diff
@@ -1,12 +1,9 @@
 FROM {{ base_image }}
 WORKDIR /app
 {% include "pre_install.j2" ignore missing %}
-{% if packages %}RUN {{ package_install_cmd }} {{ packages|join(" ")  }} {{ package_clean_cmd }}{% endif %}
-COPY requirements.txt .
-RUN pip install -r requirements.txt
-{{ mlem_install }}
+{% include "install_req.j2" %}
 {% include "post_install.j2" ignore missing %}
-COPY . ./
+{% include "copy.j2" %}
```

Comment:
why decide to put `COPY . ./` in a separate file here?

Reply:
so it can be disabled (see sagemaker build)